backend/runners/load_representative_points.py

```diff
@@ -44,17 +65,23 @@ def _main(kwargs):
     with open(kwargs['filepath'], 'r') as f:
         json_features = geojson.load(f)['features']
 
-    for feature in json_features:
-        feature['properties']['zip_code'] = FAKE_ZIP_CODE
+    if kwargs['fake_state'] or FAKE_COUNTY:
```

Comment:
Isn't this always going to evaluate to `True`? Regardless of the value of `kwargs['fake_state']`, `FAKE_COUNTY = 'county'` is at the top of the file.